tests/config_id/enc/props_kss.c

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+
+OE_SET_ENCLAVE_SGX_KSS(
+    1, /* ProductID */
+    1, /* SecurityVersion */
+    {0},
```

Comment:
Can we set ExtendedProductID / ExtendedProductID here?

Reply:
raised issue https://github.com/openenclave/openenclave/issues/4112